var/spack/repos/builtin/packages/ninja/package.py

```diff
@@ -83,5 +79,26 @@ def setup_dependent_package(self, module, dspec):
         module.ninja = MakeExecutable(
             which_string(name, path=[self.spec.prefix.bin], required=True),
             determine_number_of_jobs(parallel=dspec.package.parallel),
-            supports_jobserver=self.spec.version == ver("kitware"),
+            supports_jobserver=self.spec.version == ver("kitware") or self.name == "ninja_fortran",
```

Comment:
I don't think we should have two ways to express this. Either the vanilla ninja package should have a `kitware` version, or we should have a separate package (like you did here), but both is IMO confusing and I can't see a use case for why we'd keep it in the vanilla Ninja package.

Reply:
Note that I didn't add a way here, I added a virtual that allows for both of the ways that currently exist.  I actually agree with you that getting rid of the one in the main ninja package makes sense, but wanted to shake it out before doing that.

Reply:
Right, this was an existing bit of redundancy introduced a good while ago, these changes just make the problem worse/more obvious than it was. I actually had no idea that we even had the Kitware version of Ninja as a package in the first place.

IMO the `kitware` version in the vanilla `ninja` should probably go.